.orchestra/ci/install-dependencies.sh

```diff
@@ -20,6 +20,9 @@ apt-get install --no-install-recommends --yes \
   g++-multilib \
   gawk \
   git \
+  graphviz \
+  graphviz-dev \
+  jq \
```

Comment:
Why `jq`?

Reply:
TL:DR; ended up here by mistake but will be required soon anyway.
I'm using jq it in the CI script in the feature/non-optimized-builds branch to process a list of all the builds (for building binary archives for all builds). Maybe overkill, but structured data is always better than using bash, sed & awk IMO.

Reply:
OK, then we can import it in the appropriate commit once needed.
Drop it and I'll merge!

Reply:
Done